inst/examples/showcase/static/js/highlight_all.js

```diff
@@ -0,0 +1,3 @@
+Shiny.addCustomMessageHandler("highlight_all", function(_) {
+  hljs.highlightAll();
```

Comment:
Kudos for adding highlighting! Can we please include the version somehow? This will be useful for future developers. Either put the library in a directory with the version number, or add the version number to the file name.

Reply:
Version added in filename in #56